IfcDistributionSystem Rail Relocated Pset_EnvironmentalCondition (#312)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/a/AtmosphericPressure_0Qno0T109Cww7r$cstWuI3/Documentation.md

> Maximum level of atmospheric pressure that the equipment complies with

Reviewer: ```suggestion
Maximum level of atmospheric pressure that the equipment can operate effectively in.
```
